config/default.json

```diff
@@ -1,5 +1,4 @@
 {
     "host": "hackathon-heroku-app.herokuapp.com",
-    "path": "/occupied",
-    "auth": "1:123456"
```

Comment:
Auth should stay (at least leave it commented out). Any new user should be aware of all possible config entries.